pkgs/top-level/python-packages.nix

```diff
@@ -26943,4 +26944,41 @@ in modules // {
       license = licenses.wtfpl;
     };
   };
+
+  # tensorflow is built from a downloaded wheel, because the upstream
+  # project's build system is an arcane beast based on
+  # bazel. Untangling it and building the wheel from source is an open
+  # problem.
+
+  tensorflow = self.tensorflowNoGpuSupport;
+
+  tensorflowNoGpuSupport = buildPythonPackage rec {
+    name = "tensorflow";
+    version = "0.8.0";
+
+    src = pkgs.fetchurl {
+      url = "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${version}-cp27-none-linux_x86_64.whl";
+      sha256 = "07lb6rknngq9bicd7z1q9caiqxlqn4fdx8q24s3rqvv9wi79szws";
+    };
+
+    propagatedBuildInputs = with self; [ numpy six protobuf3_0 pkgs.swig ];
+
+    unpackPhase = ''
+      mkdir dist
+      cp $src dist/tensorflow-${version}-cp27-none-linux_x86_64.whl
+    '';
+
+    dontBuild = true;
+    doCheck = false;
+
+    preFixup = ''
+      find $out -name '*.so' -exec patchelf --set-rpath ${stdenv.lib.makeLibraryPath [ pkgs.gcc.cc.lib pkgs.zlib ]} {} \;
+    '';
+
+    meta = {
+      description = "TensorFlow helps the tensors flow (no gpu support)";
+      homepage = http://tensorflow.org;
+      license = licenses.asl20;
```

Comment:
I have no idea. To be honest, I don't know how we would typically deal with an issue like this in Nixpkgs, because it doesn't happen to often.

Maybe just add a note to it that aside from that license there are also the licenses of the bundled dependencies to consider?